frontend/src/components/BaseButton.vue

```diff
@@ -17,7 +17,7 @@
       :text-color="textColor"
```

Comment:
*arbitrary comment location*

This is *really* coming together @garyghayrat. Testing it out on Goerli and overall it's working great. I really tried to hammer on it to find edge case behavior. Here are few things I discovered:

* Once I submit the transaction and it's pending, it should no longer be possible to click the "x" to remove that row
* Same with the "+ Add Send" button. It shouldn't be possible to push this while a transaction is pending.
* After the transaction completes, the form doesn't clear. On Single send we clear the form, which I *think* makes more sense for this case too. Curious what you think @mds1.
* It's currently possible to do batch send with only 1 row. This doesn't really make sense and we should probably prevent users from doing it by preventing the send button from being clicked unless there are two valid rows.
* While a transaction is pending I can still switch back between the tabs. This seems a little odd—should we disallow it? Curious what you think.
* After completing a batch send, if I switch back to the single send tab, the form no longer has a default asset selected:

<img width="678" alt="Screen Shot 2023-05-19 at 6 04 17 PM" src="https://github.com/ScopeLift/umbra-protocol/assets/1479802/ed9ad85b-15a7-4535-85b4-3a41e724e7ff">

* Furthermore, after completing a batch send, if I switch back to the single send form and fill it out, the send button on that form remains disabled. I have to reload before I can do a single send.

Reply:
>     * Once I submit the transaction and it's pending, it should no longer be possible to click the "x" to remove that row
> 
>     * Same with the "+ Add Send" button. It shouldn't be possible to push this while a transaction is pending.
> 
>     * While a transaction is pending I can still switch back between the tabs. This seems a little odd—should we disallow it? Curious what you think.
> 

I think it makes sense to disable switching tabs while tx is pending. Disabled all mentioned during `isSending` in https://github.com/ScopeLift/umbra-protocol/pull/491/commits/bf8ce3e8d391fffd4100878a9d47f20ecb8cb18f


>     * After the transaction completes, the form doesn't clear. On Single send we clear the form, which I _think_ makes more sense for this case too. Curious what you think @mds1.
> 
>     * It's currently possible to do batch send with only 1 row. This doesn't really make sense and we should probably prevent users from doing it by preventing the send button from being clicked unless there are two valid rows.
> 
Feel like we could make a case for both, but clearing the form for now to keep the consistency in https://github.com/ScopeLift/umbra-protocol/pull/491/commits/bfcadb449f639b4623eb79072235541032fec505
 In the same commit, the `Send` button is disabled if there's only 1 row. Also now the batchSend page starts with 2 rows.

>     * After completing a batch send, if I switch back to the single send tab, the form no longer has a default asset selected:
> 
>     * Furthermore, after completing a batch send, if I switch back to the single send form and fill it out, the send button on that form remains disabled. I have to reload before I can do a single send.
These are fixed in https://github.com/ScopeLift/umbra-protocol/pull/491/commits/407ac6f90d7b5cea9c3154fd5df832724c54332e

Reply:
It's fixed here: https://github.com/ScopeLift/umbra-protocol/pull/491/commits/45c87b79f3f77595afa567a38e064c3db3e8f713